### PR TITLE
Added ACI-CNI 6.0.3.2 variables 

### PR DESCRIPTION
**Pull request by akhilesh-oc**
https://github.com/rancher/rancher/issues/43883

### CODE REVIEW

File: cluster/defaults.go [hunk 1]
```diff
@@ -902,13 +905,17 @@ func (c *Cluster) setClusterNetworkDefaults() {
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexAgentOvsAsyncjsonEnabled, DefaultAciOpflexAgentOvsAsyncjsonEnabled)
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexAgentPolicyRetryDelayTimer, DefaultAciOpflexAgentPolicyRetryDelayTimer)
 		setDefaultIfEmpty(&c.Network.AciNetworkProvider.AciMultipod, DefaultAciAciMultipod)
-		setDefaultIfEmpty(&c.Network.AciNetworkProvider.OpflexDeviceReconnectWaitTimeout, DefaultAciOpflexDeviceReconnectWaitTimeout)
```
Comment (snasovich) [inline cluster/defaults.go:905]: This removed field is still referenced by `pkg/rke/templates/aci-v6.0.3.2.go` template introduced in https://github.com/rancher/kontainer-driver-metadata/pull/1291. Do we have to remove existing fields, especially ones still referenced by the template?

@kinarashah , tagging you as you have more context on what could happen if we remove fields from types.

**Review by snasovich — CHANGES_REQUESTED** [review]

File: cluster/defaults.go (hunk 1 above)
Comment (akhilesh-oc) [reply cluster/defaults.go:905]: The variable is not removed, just the setting default parts. So other places it is still there.

**Review by snasovich — DISMISSED** [review]

**Review by kinarashah — CHANGES_REQUESTED** [review]
@akhilesh-oc Build is failing, you need to run `go generate` to update generated files due to type changes. 
``` 
 M types/kdm/zz_generated_deepcopy.go
 M types/zz_generated_deepcopy.go
 ```

**Comment by akhilesh-oc** [comment]
> @akhilesh-oc Build is failing, you need to run `go generate` to update generated files due to type changes.
> 
> ```
>  M types/kdm/zz_generated_deepcopy.go
>  M types/zz_generated_deepcopy.go
> ```

Thanks @kinarashah . Updated it and all checks are passed now. Should I squash the chnages with single commit or leave as it is?

**Review by kinarashah — DISMISSED** [review]

**Comment by snasovich** [comment]
> > @akhilesh-oc Build is failing, you need to run `go generate` to update generated files due to type changes.
> > ```
> >  M types/kdm/zz_generated_deepcopy.go
> >  M types/zz_generated_deepcopy.go
> > ```
> 
> Thanks @kinarashah . Updated it and all checks are passed now. Should I squash the chnages with single commit or leave as it is?

Since there is distinct `go generate` commit, please do NOT squash on merge.

**Review by snasovich — APPROVED** [review]

**Comment by snasovich** [comment]
@akhilesh-oc , with https://github.com/rancher/kontainer-driver-metadata/pull/1291 merged, please update KDM URL (https://github.com/rancher/rke/blob/833fec5b6d802ce57204945d51357de3d376b773/codegen/codegen.go#L13) as part of this PR and run updated `go generate`.

**Comment by akhilesh-oc** [comment]
> @akhilesh-oc , with [rancher/kontainer-driver-metadata#1291](https://github.com/rancher/kontainer-driver-metadata/pull/1291) merged, please update KDM URL (
> 
> https://github.com/rancher/rke/blob/833fec5b6d802ce57204945d51357de3d376b773/codegen/codegen.go#L13
> 
> ) to `dev-v2.7` as part of this PR and run updated `go generate`.

But the file [rke/codegen/codegen.go] in branch aci-6.0.3.2-rke-1.4, seems to be using url with 2.7

**Comment by kinarashah** [comment]
@akhilesh-oc Could you please rerun go generate? I missed that we didn't update default k8s version when introducing new versions in the KDM PR, so the build is currently failing. This is now fixed in KDM.

**Review by snasovich — APPROVED** [review]

**Pull request merged**